clients/memconfig_client/client.go

```diff
@@ -72,12 +82,18 @@ func (cfg *intermediaryCfg) IntoMemConfig() (*MemConfig, error) {
 		return nil, fmt.Errorf("failed to parse blobExpiration: %w", err)
 	}
 
+	GetReturnsInstructedStatusCode := GetReturnsInstructedStatusCode{
+		IsActivated:          cfg.GetReturnsInstructedStatusCode.IsActivated,
+		GetReturnsStatusCode: cfg.GetReturnsInstructedStatusCode.GetReturnsStatusCode,
+	}
```

Comment:
Don't think this is needed? Just use cfg.GetReturnsInstructedStatusCode directly?